portal/models/user.py

```diff
@@ -44,60 +44,23 @@
     TRUENTH_ID, TRUENTH_USERNAME) + TRUENTH_PROVIDER_SYSTEMS
 
 
-class Extension:
-    """Abstract base class for common user extension FHIR objects"""
-    __metaclass__ = ABCMeta
-
+class UserIndigenousStatusExtension(CCExtension):
+    # Used in place of us-core-race and us-core-ethnicity for
+    # Australian configurations.
     def __init__(self, user, extension):
```

Comment:
Was it an error to leave the `Extension.__init__()` method behind?  I believe you want that on the base class.